### PR TITLE
Create funding schema

### PR DESCRIPTION
**Pull request by skoehnen**
For issue #85

### CODE REVIEW

File: v3.0/products/funding.schema.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "required": [
+    "@type",
+    "@id",
+    "funderName"
+  ],
+  "properties": {
+    "@type": {
+      "description": "Metadata node type (schema).",
+      "type": "string",
+      "const": "https://openminds.ebrains.eu/core/contribution"
```
Comment (lzehl) [inline v3.0/products/funding.schema.json]: needs to be "..../core/funding"

File: v3.0/products/funding.schema.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    "awardNumber": {
+      "description": "Number of this awarded funding.",
+      "type": "string"
+    },
+    "awardTitle": {
+      "description": "Title of this awarded funding.",
+      "type": "string"
+    },
+    "funderName": {
```
Comment (lzehl) [inline v3.0/products/funding.schema.json]: I know I did use "funderName" before, but it does seem weird to me now that a property with NAME in it is not a free text. Could you replace it with "funder" ?

Description: "Link to the funding organization"

**Review by lzehl — CHANGES_REQUESTED** [review]

File: v3.0/products/funding.schema.json (hunk 1 above)
Comment (skoehnen) [reply v3.0/products/funding.schema.json]: I thought that as well, but you specified `contribution` in #85 

But that is no big problem I can change it 😉

File: v3.0/products/funding.schema.json (hunk 2 above)
Comment (skoehnen) [reply v3.0/products/funding.schema.json]: Sure

Comment (skoehnen) [reply v3.0/products/funding.schema.json]: I changed

File: v3.0/products/funding.schema.json (hunk 1 above)
Comment (skoehnen) [reply v3.0/products/funding.schema.json]: I updated the const, is this okay that way?

**Comment by lzehl** [comment]
Looks good to me. thanks

**Pull request merged**